events-api/src/modules/tickets/dto/create-ticket.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNumber, IsString, IsUUID, Min } from "class-validator";
+
+export class CreateTicketReqDto {
+  @IsNumber()
+  @Min(0)
```

Comment:
Use IsPositive decorator instead, also check if it an integer and not a float

Reply:
Done